### PR TITLE
Silly Slimes

### PR DESCRIPTION
**Pull request by fenndragon**
# Description

Changed the Sprites of 4 particular slimes

---

# Changelog

:cl:

-Changed 4 of the elemental sprites

### CODE REVIEW

File: Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "version": 1,
+  "license": "CC0-1.0",
+  "copyright": "Created by TheShuEd (github) for Space Station 14.",
```
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json:4]: Please state the edit you did to the sprites.

Example: Edited by ...

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Should those sprites not be applied to the non-floof slimes also?

**Review by FoxxoTrystan — APPROVED** [review]

File: Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json:1]: You need to reffer to anyone that did a change.

Orignal creator of the sprite + edited by ...

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Please change the changlog, those seem to relate to all slimes.

File: Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json
Comment (fenndragon) [reply Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json:1]: *blinks* no, I specified that I changed 4 slimes sprites, there are only 2 sprites. 

there are 4 slimes that get those 2 sprites. it's pretty clear I didn't say I was changing all slimes

Comment (fenndragon) [reply Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json:1]: also OH! we're doing credits on this stuff now? I... don't actually think I've put credits on any of the work I've done so far...

Comment (FoxxoTrystan) [reply Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json:1]: You have to mention that you edited the sprites.

Comment (fenndragon) [reply Resources/Textures/Floof/Mobs/Animals/elemental.rsi/meta.json:1]: where do I do this?

File: Resources/Textures/Mobs/Aliens/Xenos/rouny.rsi/meta.json
Comment (FoxxoTrystan) [inline Resources/Textures/Mobs/Aliens/Xenos/rouny.rsi/meta.json:1]: This change is not related to the PR.
But il accept it.

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Memeji — APPROVED** [review]

**Pull request merged**